Api/Services/UserService.cs

```diff
@@ -196,4 +197,95 @@ public async Task<List<UserEmployeeVM>> GetEmployeesAsync(string userId)
             })
             .ToListAsync();
     }
+
+    /// <summary>
+    /// Gets the employee with the given id, provided he works for the current user
+    /// </summary>
+    /// <param name="empId"></param>
+    /// <param name="user"></param>
+    /// <returns></returns>
+    public async Task<Result<User>> GetEmployeeAsync(string empId, ClaimsPrincipal user)
+    {
+
+        var errors = new List<ValidationResult>();
+
+        var owner = (await userManager.GetUserAsync(user))!;
+        var emp = await userManager.FindByIdAsync(empId);
+
+        if (emp is null)
+        {
+            errors.Add(new ValidationResult($"Emp: {empId} not found", [nameof(empId)]));
```

Comment:
Tu podajesz nazwę parametru (`empId`), dlatego końcówka zwróci ją w responsie. Ale np. ten parametr w końcówce nazywa się `employeeId`, a nawet jakby i nazywał się tak samo, to nie mielibyśmy sposobu zapewnić to, że nie pomylimy się w przyszłości. Dlatego moim zdaniem nie warto podawać tu drugiego parametru (`[nameof(empId)]`), wtedy w responsie będzie po prostu `""`.